### PR TITLE
docs: unify contribution guidelines 

### PR DESCRIPTION
**Pull request by mvidalgarcia**
Closes https://kiwicom.atlassian.net/browse/FEPLT-1667

### CODE REVIEW

**Review by DSil — APPROVED** [review]
👌 🚀

**Comment by mvidalgarcia** [comment]
I've just realised there's another guide pretty similar to this one:

https://github.com/kiwicom/orbit/blob/master/.github/contributing.md

**Comment by DSil** [comment]
The one you edit on this PR is the one referenced on the README of the repo. I don't know the use purpose of the one you mentioned now 🤔

**Comment by mvidalgarcia** [comment]
Yes, that's how I reached it. But then the GitHub UI displayed a popup saying that the contribution guidelines changed since my last contribution, and linked to that 😅 

I see that also shows up when creating a new issue.

image

And it's the recommended way, so perhaps we should get rid of `.github/contribution/README.md` and move it to `.github/contributing.md`.

**Comment by DSil** [comment]
Yes, IMO we should merge into just one, making sure everywhere links to the correct place

**Comment by mvidalgarcia** [comment]
I unified both guides in one, under `.github/CONTRIBUTING.md`

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+```
+
+Build the design tokens and the tailwind preset:
+
+```sh
+yarn tokens build
+yarn tailwind-preset build
+```
+
+For developing Orbit, components run Storybook:
```
Comment (DSil) [inline .github/CONTRIBUTING.md:93]:
```suggestion
For developing Orbit components, run Storybook:
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+```sh
+yarn dev
+```
+
+If you would like to try out the development version of one of our packages, we recommend using [yalc](https://github.com/wclr/yalc):
+
+```sh
+yarn global add yalc
+```
```
Comment (DSil) [inline .github/CONTRIBUTING.md:103]: This refers on how to install a global dependency. Should we be responsible for documenting it? Or at least say something like
> If you don't have it installed, consider installing it globally: `yarn global add yalc`

💭  just a suggestion, non-blocking.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+We want to ensure that all props are consistent across all components. [This convention](contribution/props-convention) can help choose the right prop names in for your new components.
+
+## Component design
+
+Check [this section](contribution/component-design) for a complete description of the structure of the files which are necessary for a Component check.
+
+## Commits
+
+We are using [this commits convention](contribution/commits) to ensure that all commits are pushed in some logic from all devs.
```
Comment (DSil) [inline .github/CONTRIBUTING.md:166]: I don't think "in some logic from all devs" is what we want to say here. Can you rephrase it? 🙏

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+## Development Information
+
+### Requirements
+
+- [Node 14.13+](https://nodejs.org/en/)
+- [Yarn](https://yarnpkg.com/en/)
+
+### Technologies we use
+
+- [React](https://reactjs.org/docs/getting-started.html) for development
```
Comment (DSil) [inline .github/CONTRIBUTING.md:38]:
```suggestion
- [React](https://react.dev) for development
```

**Review by DSil — CHANGES_REQUESTED** [review]
A few suggestions since we are touching this file.

But blocking because all the internal links are broken 🚨

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (mvidalgarcia) [reply .github/CONTRIBUTING.md:166]: Definitely 😅 I copied it blindly from the other doc.

**Review by DSil — APPROVED** [review]
